### PR TITLE
fix(space-nuxt-base): upgrade app-extension-auth

### PR DESCRIPTION
**Pull request by eunjae-lee**
## What?

This PR upgrads app-extension-auth to v2.

### CODE REVIEW

File: space-plugins/nuxt-base/server/middleware/02.auth.global.ts
Comment (BibiSebi) [inline space-plugins/nuxt-base/server/middleware/02.auth.global.ts:21]: The `init_oauth` is currently not being used in the app-extension auth right? Should we still forward it?

Comment (eunjae-lee) [reply space-plugins/nuxt-base/server/middleware/02.auth.global.ts:21]: true. let's not!

**Review by BibiSebi — APPROVED** [review]
Let's go :)

File: space-plugins/nuxt-base/server/utils/getAppSession.ts
Comment (BibiSebi) [inline space-plugins/nuxt-base/server/utils/getAppSession.ts:36]: maybe we can even expose isAppSession validation method which we could use here?

**Review by BibiSebi — APPROVED** [review]
Looks very neat.

**Comment by BibiSebi** [comment]
nned to wait for next release as inferSession function is not released yet

**Comment by BibiSebi** [comment]
Everything is updated, but still waiting before we finalise the changes on app-extension-auth before merging this.

File: space-plugins/nuxt-base/server/utils/getAppSession.ts
Comment (eunjae-lee) [reply space-plugins/nuxt-base/server/utils/getAppSession.ts:36]: that's a good point.

**Pull request merged**
